apps/src/templates/sectionAssessments/FreeResponsesSurveyContainer.jsx

```diff
@@ -6,7 +6,7 @@ import {
 import { connect } from 'react-redux';
 import i18n from "@cdo/locale";
 
-const QUESTION_CHARACTER_LIMIT = 260;
+const QUESTION_CHARACTER_LIMIT = 110;
```

Comment:
tiny Q: are the surveys and assessments closely related enough that this should a shared constant?

Reply:
Will follow up in a separate PR.